### DISCUSSION TITLE
logfile Error make connection

### DISCUSSION THREAD

**Discussion by melmager** [dianlight/hassio-addons#280] [Q&A] [answered]
Log file entry
create_connection_session_info: guest user (from session setup) not permitted to access this share (ATA_S11_512G_PHISON_SSD_B27_D839070B130800417215_PART1)
make_connection_snum: create_connection_session_info failed: NT_STATUS_ACCESS_DENIED

moredisk
id:ata-S11-512G-PHISON-SSD-B27_D839070B130800417215-part1

[10:46:58] INFO: Exposed Disks Summary:
[CONFIG]                path = /homeassistant # TM:false US:homeassistant #
[ADDONS]                path = /addons # TM:false US:homeassistant #
[SSL]                   path = /ssl # TM:false US:homeassistant #
[SHARE]                 path = /share # TM:false US:homeassistant #
[BACKUP]                path = /backup # TM:false US:homeassistant #
[MEDIA]                 path = /media # TM:false US:homeassistant #
[ADDON_CONFIGS]         path = /addon_configs # TM:false US:homeassistant #
[ATA_S11_512G_PHISON_SSD_B27_D839070B130800417215_PART1]    path = /mnt/ata-S11-512G-PHISON-SSD-B27_D839070B130800417215-part1 # TM:true US:homeassistant #

using as /backup in hass

what am i missing ?

**Comment by dianlight** [comment, marked as answer]
Is difficulty to understand your problem without a context and your configuration.

From the log I can suppose you are trayng to access from a pc without 
setting username and password.

create_connection_session_info: guest user (from session setup) not permitted to access this share
